### ISSUE TITLE
[0.4] Add version info to clusterawsadm

### ISSUE THREAD

**Issue by erwinvaneyk** [kubernetes-sigs/cluster-api-provider-aws#1237]
/kind bug

The binaries included in the releases do not seem contain versioning info. A minor issue, but it makes it a bit annoying to see which version of clusterawsadm is installed.

For example, when using clusterawsadm from 0.4.3:
```
$ clusterawsadm version
Version Info: GitReleaseTag: "", MajorVersion: "", MinorVersion:"", GitReleaseCommit:"", GitTreeState:""
```

**Environment:**

- Cluster-api-provider-aws version: tested with both 0.4.3 and 0.4.2
- OS (e.g. from `/etc/os-release`): Darwin

**Comment by ncdc** [comment]
Need to backport #1232 to the release-0.4 branch

**Comment by detiber** [comment]
It looks like this has landed in both master and release-0.4 now, so closing.
/close
